src/asset.rs

```diff
@@ -726,7 +767,8 @@ impl AssetPool {
         &self.active
     }
 
-    /// Decommission assets whose lifetime has passed and commission new assets
+    /// Decommission assets whose lifetime has passed or have been unused for too long,
```

Comment:
```suggestion
    /// Decommission assets whose lifetime has passed,
```
Actually this can go back to how it was as we're not doing this here!